packages/extension-polkagate/src/popup/account/unlock/Review.tsx

```
@@ -10,27 +9,28 @@
  * */
 
 import type { ApiPromise } from '@polkadot/api';
+import type { SubmittableExtrinsic } from '@polkadot/api/types';
+import type { Chain } from '@polkadot/extension-chains/types';
+import type { Balance } from '@polkadot/types/interfaces';
+import type { ISubmittableResult } from '@polkadot/types/types';
+import type { BN } from '@polkadot/util';
+import type { Lock } from '../../../hooks/useAccountLocks';
+import type { Proxy, ProxyItem, TxInfo } from '../../../util/types';
```

Biome

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)